platform/security/servlet/security-servlet-session-expiry/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:jaxrs="http://cxf.apache.org/blueprint/jaxrs"
+           xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+
+    <reference id="securityManager" interface="ddf.security.service.SecurityManager"/>
+
+    <bean id="sessionService"
+          class="org.codice.ddf.security.servlet.expiry.SessionManagementService">
+        <property name="securityManager" ref="securityManager"/>
+    </bean>
+
+    <jaxrs:server id="restService" address="/session">
```

Comment:
❓ Do we want to make it clear this is really only for internal use?  If so then we should consider moving this to `/internal/session`.